### PR TITLE
Add TO client DS ByServer, Regexes

### PR DESCRIPTION
**Pull request by rob05c**
Adds missing CDN DomainName field

### CODE REVIEW

File: traffic_ops/client/delivery_service_endpoints.go [hunk 1]
```diff
@@ -22,6 +22,10 @@ func deliveryServicesEp() string {
 	return apiBase + dsPath + ".json"
 }
 
+func deliveryServicesByServerEp(id string) string {
+	return apiBase + "/servers/" + id + dsPath + ".json"
```
Comment (mitchell852) [inline traffic_ops/client/delivery_service_endpoints.go]: you don't need the .json anymore on any api routes btw

Comment (rob05c) [inline traffic_ops/client/delivery_service_endpoints.go]: Right, I was just trying to be consistent with the surrounding functions

Comment (mitchell852) [inline traffic_ops/client/delivery_service_endpoints.go]: consistency is good

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-traffic_ops-test-PR/13/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/13/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/381/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/382/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/383/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/338/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/339/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/340/

**Pull request merged**